pom.xml

```diff
@@ -67,6 +67,11 @@
             <artifactId>mapstruct</artifactId>
             <version>${mapstruct.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+            <version>3.1.2</version>
```

Comment:
you don't need to specify version here - it is taken from `spring-boot-starter-parent`